x-pack/plugins/canvas/public/components/var_config/var_config.tsx

```diff
@@ -0,0 +1,222 @@
+  const [selectedVar, setSelectedVar] = useState<CanvasVariable | null>(null);
+
+  const selectAndEditVar = (v: CanvasVariable) => {
+    setSelectedVar(v);
+    setPanelMode(PanelMode.Edit);
+  };
+
+  const selectAndDeleteVar = (v: CanvasVariable) => {
+    setSelectedVar(v);
+    setPanelMode(PanelMode.Delete);
+  };
+
+  const actions = [
+    {
+      name: strings.getCopyActionButtonLabel(),
+      description: strings.getCopyActionTooltipLabel(),
+      icon: 'copyClipboard',
+      type: 'icon',
+      isPrimary: true,
+      onClick: onCopyVar,
+    },
+    {
+      name: strings.getEditActionButtonLabel(),
+      description: '',
+      icon: 'pencil',
+      type: 'icon',
+      onClick: selectAndEditVar,
+    },
+    {
+      name: strings.getDeleteActionButtonLabel(),
+      description: '',
+      icon: 'trash',
+      type: 'icon',
+      color: 'danger',
+      onClick: selectAndDeleteVar,
+    },
+  ];
+
+  const varColumns = [
+    {
+      field: 'type',
+      name: strings.getTableTypeLabel(),
+      sortable: true,
+      render: (varType: 'string' | 'boolean' | 'number', _v: CanvasVariable) => {
+        return <EuiToken iconType={typeToToken[varType]} />;
+      },
+      width: '50px',
+    },
+    {
+      field: 'name',
+      name: strings.getTableNameLabel(),
+      sortable: true,
+    },
+    {
+      field: 'value',
+      name: strings.getTableValueLabel(),
+      sortable: true,
+      truncateText: true,
+    },
+    {
+      actions,
+      width: '60px',
+    },
+  ];
+
+  return (
+    <div
+      className={`canvasArg--expandable canvasVarConfig__container ${
+        panelMode !== PanelMode.List ? 'canvasVarConfig-isEditMode' : ''
+      }`}
+    >
+      <div className="canvasVarConfig__innerContainer">
+        <EuiAccordion
+          id="accordion-variables"
+          className="canvasVarConfig__listView canvasArg__accordion"
+          buttonContent={
+            <EuiToolTip content={strings.getTitle()} position="left" className="canvasArg__tooltip">
+              <EuiText size="s" color="subdued">
+                {strings.getTitle()}
+              </EuiText>
+            </EuiToolTip>
+          }
+          extraAction={
+            <EuiToolTip position="top" content={strings.getAddTooltipLabel()}>
+              <EuiButtonIcon
+                color="primary"
+                iconType="plusInCircle"
+                onClick={() => {
+                  setSelectedVar(null);
+                  setPanelMode(PanelMode.Edit);
+                }}
+              />
+            </EuiToolTip>
+          }
+        >
+          {variables.length !== 0 && (
+            <div className="canvasArg__content">
+              <EuiInMemoryTable
+                className="canvasVarConfig__list"
+                items={variables}
+                columns={varColumns}
+                hasActions={true}
+                pagination={false}
+                sorting={true}
+              />
```

Comment:
I'm not sure if it's possible with a prop (couldn't find one), but curious if we have the ability to remove/disable the table row highlighting. It doesn't look so hot on a gray background and it's largely unnecessary on such a narrow table.